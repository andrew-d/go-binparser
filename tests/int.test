[
    {"name": "IntTest",
     "fields": [
        {"name": "f_int8", "offset": 0, "type": "int8"},
        {"name": "f_int16", "offset": 1, "type": "int16"},
        {"name": "f_int32", "offset": 3, "type": "int32"},
        {"name": "f_int64", "offset": 7, "type": "int64"}
    ]}
]
