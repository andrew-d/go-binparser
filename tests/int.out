F_Int8,%!s(int8=-1)
F_Int16,%!s(int16=-1)
F_Int32,%!s(int32=-1)
F_Int64,%!s(int64=-1)
